transformer_engine/paddle/distributed.py

```diff
@@ -54,6 +59,19 @@ def get_tp_group_and_world_size(
     return model_parallel_group, world_size
 
 
+def is_pp_enabled() -> int:
+    if not (paddle.distributed.is_initialized()):
+        return 1
+
+    return tp._HYBRID_PARALLEL_GROUP.get_pipe_parallel_world_size()
```

Comment:
Based on the name and usage in `TransformerEngineBaseLayer`:
```suggestion
def is_pp_enabled() -> bool:
    if not paddle.distributed.is_initialized():
        return False
    return tp._HYBRID_PARALLEL_GROUP.get_pipe_parallel_world_size() > 1
```
If the implementation is intended, the name should be changed to something like `get_pipeline_parallel_size` to avoid confusion.

Reply:
Thank you for pointing out the mistake. I have corrected it accordingly.